File: Iniciante/Lua/1010.lua
-- lê os valores da primeira peça
local codigo1, num1, valor1 = io.read("*n", "*n", "*n")
-- lê os valores da segunda peça
local codigo2, num2, valor2 = io.read("*n", "*n", "*n")

-- calcula o valor total a ser pago
local total = num1 * valor1 + num2 * valor2

-- imprime o resultado
io.write(string.format("VALOR A PAGAR: R$ %.2f\n", total))

File: Iniciante/Python/1010.py
# Ler os dados da peça 1
codigo_peca1, numero_peca1, valor_unitario1 = map(float, input().split())

# Ler os dados da peça 2
codigo_peca2, numero_peca2, valor_unitario2 = map(float, input().split())

# Calcular o valor total a ser pago
total = (numero_peca1 * valor_unitario1) + (numero_peca2 * valor_unitario2)

# Imprimir o resultado
print("VALOR A PAGAR: R$ {:.2f}".format(total))
